ui/app/AppLayouts/Wallet/controls/EditSlippagePanel.qml

```diff
@@ -0,0 +1,107 @@
+    QtObject {
+        id: d
+        readonly property string selectedToTokenSymbol: !!root.selectedToToken && !!root.selectedToToken.symbol ?
+                                                   root.selectedToToken.symbol : ""
+    }
+
+    horizontalPadding: Style.current.padding
+    verticalPadding: Style.current.bigPadding
+
+    background: Rectangle {
+        radius: 16
+        border.width: 1
+        border.color: Theme.palette.directColor8
+        color: Theme.palette.indirectColor3
+    }
+
+    contentItem: ColumnLayout {
+        id: baseLayout
+        spacing: Style.current.bigPadding
+        RowLayout {
+            StatusBaseText {
+                Layout.fillWidth: true
+                text: qsTr("Slippage tolerance")
+                font.weight: Font.Medium
+                lineHeight: button.implicitHeight
+                lineHeightMode: Text.FixedHeight
+                verticalAlignment: Text.AlignVCenter
+            }
+            StatusLinkText {
+                id: button
+                visible: slippageSelector.isEdited
+                text: qsTr("Use default")
+                normalColor: Theme.palette.primaryColor1
+                onClicked: slippageSelector.reset()
+            }
+        }
+        StatusBaseText {
+            Layout.fillWidth: true
+            Layout.topMargin: -12
+            text: qsTr("Maximum deviation in price due to market volatility and liquidity allowed before the swap is cancelled. (0.5% default).")
+            wrapMode: Text.Wrap
+            color: Theme.palette.directColor5
+        }
+        /* TODO: error conditions for custom enteries missing will be done under -
+        https://github.com/status-im/status-desktop/issues/15017 */
+        SlippageSelector {
+            id: slippageSelector
+            objectName: "slippageSelector"
+            Layout.fillWidth: true
+        }
+        StatusModalDivider {
+            Layout.fillWidth: true
+            Layout.topMargin: 0
+            Layout.bottomMargin: -Style.current.smallPadding
+            dividerColor: Theme.palette.directColor8
+        }
+        RowLayout {
+            spacing: 4
+            StatusBaseText {
+                text: qsTr("Receive at least")
+                font.pixelSize: 13
+                font.weight: Font.Medium
+            }
+            StatusSmartIdenticon {
+                Layout.preferredWidth: Style.current.padding
+                Layout.preferredHeight: Style.current.padding
+                asset.name: !!root.selectedToToken && !!root.selectedToToken.image
+                              ? root.selectedToToken.image
+                              : Constants.tokenIcon(d.selectedToTokenSymbol)
+                asset.isImage: true
+            }
+            StatusTextWithLoadingState {
+                text: {
+                    let amount = parseFloat(root.toTokenAmount)
+                    let percentageAmount = (amount - ((amount/100) * slippageSelector.value))
```

Comment:
hmm we should probably cap the slippage value, it's silly to show negatives here if the user enters some crazy high value.
Actually, we should probably only show results here when we're not showing some error state due to the entered custom amount. Something to ask Ben perhaps?

Reply:
@dlipicar this is already handled in the UI and in design we let the user enter it, but we show an error message below and block swapping. The error message is missing in the SlippageSelector and Ive raised a bug for some issues I found in it https://github.com/status-im/status-desktop/issues/15017, so after that is addresses, it will be more clear that it doesnt make sense to the user